Content.Shared/Contests/ContestsSystem.CVars.cs

```diff
@@ -0,0 +1,19 @@
+using Content.Shared.CCVar;
+using Robust.Shared.Configuration;
+
+namespace Content.Shared.Contests
+{
+    public sealed partial class ContestsSystem
+    {
+        [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+        public bool DoMassContests { get; private set; }
+        public float MassContestsMaxPercentage { get; private set; }
+
+        private void InitializeCVars()
+        {
+            Subs.CVar(_cfg, CCVars.DoMassContests, value => DoMassContests = value, true);
+            Subs.CVar(_cfg, CCVars.MassContestsMaxPercentage, value => MassContestsMaxPercentage = value, true);
+        }
+    }
+}
```

Comment:
```suggestion
```

Reply:
The fuck? Why are you asking me to delete these?

Reply:
Use the CVars normally

Reply:
What do you mean **normally**, this is the normal way they're used. What the hell?

Reply:
We've talked about this before

Reply:
Use `IConfigurationManager.GetCVar`